### DISCUSSION TITLE
Spinnaker version 1.29.0.5

### DISCUSSION THREAD

**Discussion by eintzandt** [bonsai-rx/bonsai#1337] [Q&A] [answered]
Hi all, 

I want to use FLIR cameras with bonsai but it requires version 1.29.0.5. I can't find the link to download it anywhere, does anyone have it? (this link that others have posted is unavailable to me: https://flir.app.boxcn.net/v/SpinnakerSDK/folder/113206835923). Alternatively, does anyone have experience using the spinnaker github package so that I can keep using a newer version of Spinnaker?

Thank you!

**Comment by bruno-f-cruz** [comment]
Thanks for the heads up. We are trying to figure out a long term solution! Meanwhile, the current solution would be to email the company asking for this specific version, unfortunately their license does not allow 3rd parties to distribute the drivers...

**Comment by bruno-f-cruz** [comment, marked as answer]
Good news,
It seems they made it available [here](https://www.flir.com/support-center/iis/machine-vision/downloads/spinnaker-sdk-download/spinnaker-sdk--download-files/#anchor4)

You will need to register to download it.
B

**Comment by vigji** [comment]
Here for the same issue, in the linked page I think I can find only versions > 2.x even when looking in the "previous versions" section. I guess e-mailing them is the only solution atm?

**Reply by bruno-f-cruz** [reply]
I can still see them at the end of the page

**Reply by vigji** [reply]
Thanks! My bad, got lost after authentication :)
